controllers/operands/kubevirt.go

```diff
@@ -446,6 +452,7 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 		SeccompConfiguration:         seccompConfig,
 		EvictionStrategy:             hc.Spec.EvictionStrategy,
 		KSMConfiguration:             hc.Spec.KSMConfiguration,
+		VMRolloutStrategy:            &rolloutStrategy,
```

Comment:
Please use ptr, and remove the `rolloutStrategy` var; e.g. 
```suggestion
		VMRolloutStrategy:            ptr.To(kubevirtcorev1.VMRolloutStrategyLiveUpdate)
```